src/Sarif/Autogenerated/Suppression.cs

```diff
@@ -46,6 +47,14 @@ public SarifNodeKind SarifNodeKind
         [JsonConverter(typeof(Microsoft.CodeAnalysis.Sarif.Readers.EnumConverter))]
         public SuppressionKind Kind { get; set; }
 
+        /// <summary>
+        /// A string that indicates the state of the suppression. Well-known values are `accepted`, `underReview` and `rejected`.
+        /// </summary>
+        [DataMember(Name = "state", IsRequired = false, EmitDefaultValue = false)]
+        [DefaultValue("accepted")]
```

Comment:
>accepted [](start = 23, length = 8)

A default value of "accepted" is odd to me (I realize it's specified in the bug.) But "accepted" implies that someone looked at the issue and declared it "accepted". Shouldn't the default be empty or "undefined".

Reply:
thanks Ryley - leaving this PR open till we have a  final decision on the above.

Larry's suggestion was

Define two values, unknown and notReviewed. The default is notReviewed, which makes sense for direct producers. However, in case of converters (where we dont know what the original intention was), it can set the value to unknown.


@mikefan@microsoft.com what are you thoughts on this?


---
In reply to: [277044665](https://github.com/Microsoft/sarif-sdk/pull/1442#discussion_r277044665) [](ancestors = 277044665)